File: models/action.py
class Action:

    # действие субъекта над объектом

    def __init__(self, name, goodness, id):
        self.name = name      # название действия
        self.goodness = goodness  # на сколько действие доброе 0 - нейтральное
        self.id = id

File: Noosphere/models/Action.cs
using System;

namespace BrocaZone.models;

public class Action
{
    public Action(string name, Int32 goodness, Int32 id)
    {
        Id = id;
        Name = name;
        Goodness = goodness;
    }

    public Int32 Id { get; set; }
    public string Name { get; set; }
    public Int32 Goodness { get; set; }
}
